packages/jest-runtime/src/index.ts

```diff
@@ -887,6 +888,42 @@ class Runtime {
       return this._environment.global.process;
     }
 
+    if (moduleName === 'module') {
+      const createRequire = (modulePath: string | URL) => {
+        const filename =
+          typeof modulePath === 'string'
+            ? modulePath
```

Comment:
people can pass `file:///stuff` right? We might need to coerce to absolute file path